spec/app/lib/bing_api_v7_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+require 'json'
```

Comment:
remove this

Reply:
NameError:
  uninitialized constant BingApiV7

Reply:
remove only `require 'json'` !